packages/codegen/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "joist-codegen",
-  "version": "0.1.0-bump",
+  "version": "1.0.0-bump",
```

Comment:
This would end up being `1.0.537`. As we want to bump majors/minors, we'd change the `major`/`minor` vars in `bump.sh`.